test/built-ins/RegExp/named-groups/non-unicode-property-names-invalid.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 the Apple Inc. All rights reserved.
```

Comment:
"the Apple Inc." sounds a little weird; is this the correct license header?

Reply:
I've borrowed it from other test files authored by JSC team:
* [`/test/built-ins/Object/prototype/toString/proxy-function.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Object/prototype/toString/proxy-function.js)
* [`/test/built-ins/Object/prototype/toString/symbol-tag-non-str-proxy-function.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Object/prototype/toString/symbol-tag-non-str-proxy-function.js)

Reply:
Given that those are the only two hits for "the Apple Inc." and other files use "Apple Inc.", I'm assuming these are mistakes.

Reply:
Eliminate "the" and just use "Apple Inc.".

Reply:
https://www.youtube.com/watch?v=PEgk2v6KntY

Reply:
https://www.youtube.com/watch?v=PEgk2v6KntY